docs/swarm/swarm-tutorial/add-nodes.md

```diff
@@ -23,24 +23,35 @@ This tutorial uses the name `worker1`.
 [Create a swarm](create-swarm.md) tutorial step to create a worker node joined to the existing swarm:
 
     ```bash
-    $ docker swarm join --secret 4ao565v9jsuogtq5t8s379ulb \
-      --ca-hash sha256:07ce22bd1a7619f2adc0d63bd110479a170e7c4e69df05b67a1aa2705c88ef09 \
-      192.168.99.100:2377
+    $ docker swarm join \
+    --token SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c \
+    192.168.99.100:2377
```

Comment:
really nit: can you indent the second, and third line (like the previous version)?